src/components/snapshot/overview/SnapshotExport.tsx

```diff
@@ -73,16 +72,19 @@ function SnapshotExport(props: SnapshotExportProps) {
     terraUrl,
     userRoles,
   } = props;
-  const exportResponseManifest =
-    exportResponse &&
-    exportResponse.format &&
-    exportResponse.format.parquet &&
-    exportResponse.format.parquet.manifest;
+  const exportResponseManifest = exportResponse?.format?.parquet?.manifest;
```

Comment:
This looked like a place that we could leverage optional chaining instead.

Reply:
good call!